dev-db/tokyocabinet/tokyocabinet-1.4.48-r3.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools toolchain-funcs
+
+DESCRIPTION="A library of routines for managing a database"
+HOMEPAGE="https://fallabs.com/tokyocabinet/"
+SRC_URI="https://fallabs.com/tokyocabinet/${P}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~loong ~mips ~ppc ~ppc64 ~riscv ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x64-solaris"
+IUSE="bzip2 debug doc examples test threads zlib"
+RESTRICT="!test? ( test )"
+
+DEPEND="
+	bzip2? ( app-arch/bzip2 )
+	zlib? ( sys-libs/zlib )
+"
+RDEPEND="${DEPEND}"
+
+PATCHES=( "${FILESDIR}/fix_rpath.patch" )
+
+src_prepare() {
+	default
+
+	sed \
+		-e "/ldconfig/d" \
+		-e "/DATADIR/d" \
+		-i Makefile.in || die
+
+	# cflags fix - remove -O2 at end of line and -fomit-frame-pointer
+	sed -i -e 's/-O3"$/"/' configure.in || die
```

Comment:
Either the comment or the sed expression is wrong.  Is it -O2 or -O3?

Reply:
I think the previous maintainer would like to leave the end user free to decide to use -O2 or -O3 instead of forcing -O3 by default